spikeinterface/combinato/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM spikeinterface/combinato-base:0.1.0
+
+RUN pip install simplejson requests click
+
+#########################################
+### SpikeInterface
+RUN pip install spikeextractors==0.9.6 spiketoolkit==0.7.5 spikesorters==0.4.4
+
+### Install extractors
+RUN pip install h5py
+RUN pip install scipy
+RUN pip install pyintan
+RUN pip install pyopenephys
+RUN pip install neo
+RUN pip install MEArec
+RUN pip install pynwb
+RUN pip install lxml
+RUN pip install nixio
+RUN pip install shybrid pyyaml
+RUN pip install mtscomp
+RUN pip install sonpy
```

Comment:
If we modify this list, is there a way to not copy/paste this everywhere ?

I think we should do stuff like
```pip install spikeinterface[full]```
or 
```pip install spikeinterface[extended]```

and the full list would be handle directlty in spikeinetrface.
It would be better no ?

Reply:
That's a great idea Sam. Do you know how to do it?

Reply:
It is very simple with `extras_require` in setup.py

Reply:
Yeah looks very easy :)